.github/workflows/c_actions.yml

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2020 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# This workflow will install Python dependencies, run tests, lint and rat with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: C Actions
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Checkout SupportScripts
+      uses: actions/checkout@v2
+      with:
+        repository: SpiNNakerManchester/SupportScripts
+        path: support
+        
+    - name: Set SPINN_DIRS
```

Comment:
This step ought to be combined with the next one really.

Reply:
It might be an idea to echo what `SPINN_DIRS` actually is set to.

Reply:
The two steps can't be combined because of the way that using $GITHUB_ENV works - the environment variables are only set *after* the current step.  So the following variables need to be in a separate step in order to use SPINN_DIRS properly.

Reply:
You can also set environment variables for the current step; they just don't persist.

Reply:
We can always tinker with this later.